src/ros_filter.cpp

```diff
@@ -1135,7 +1134,7 @@ void RosFilter::loadParams()
 
         topic_subs_.push_back(node_->create_subscription<
             geometry_msgs::msg::PoseWithCovarianceStamped>(
-            pose_topic, pose_callback));
+            pose_topic, 10, pose_callback));
```

Comment:
I think this should be the `_queue_size` parameter depth. Same for all following uses of the same thing (so I dont make 10 comments of the same)

Reply:
Thanks, yeah I have a bunch of changes coming in, so feel free to put off the review for just a while and I'll make it more complete. The PR was a bit pre-mature

Reply:
Well small changes are good. I think most maintainers would prefer 10 small PRs vs 1 megaPR that does everything under the sun & hard to just visually check and say "yup looks good" vs needing extensive testing for how much content changed.

Reply:
This is just parameterization declaration changes, EZ to just say "yup still compiles **merge**"

Reply:
I def agree, so maybe I will rename this PR to reflect what the major thing is: getting navsat_transform_node to work.

Reply:
> I think this should be the `_queue_size` parameter depth. Same for all following uses of the same thing (so I dont make 10 comments of the same)

I'll get it fixed, thanks!